.github/workflows/release_pg_slim.yml

```diff
@@ -0,0 +1,36 @@
+name: Release for Postgres slim
+
+on:
+  workflow_dispatch:
+
+jobs:
+  pg-slim:
+    strategy:
+      matrix:
+        version: [14, 15, 16, 17]
+        platform: ["amd64"]
+    runs-on: ubuntu-latest
+    env:
+      PG_MAJOR: ${{ matrix.version }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERIO_MODELZ_USERNAME }}
+          password: ${{ secrets.DOCKERIO_MODELZ_TOKEN }}
+      - name: Push binary release to Docker Registry
+        uses: docker/build-push-action@v4
+        with:
+          context: ./docker/pg-slim
+          push: true
+          platforms: "linux/${{ matrix.platform }}"
+          file: ./docker/pg-slim/Dockerfile
+          build-args: |
+            PG_MAJOR=${{ matrix.version }}
+          tags: modelzai/pg-slim:${{ matrix.version }}-${{ matrix.platform }}
```

Comment:
No need, this is basically immutable.

Reply:
Why do we need the pg_slim in this repo since it already exists in pgvecto.rs?

Reply:
We can only maintain it in this repository, the pgvecto.rs repository will be deleted.

Reply:
What does pg-slim do? It looks like a compiled postgres for docker usage. I think it's not related to the extension and will not be released with the extension. Can we create another repo or use the cnpg repo?

Reply:
We need to specify the `datarootdir` and `libdir` in mutable storage during the Postgres building, as the lib folder in our cluster is mounted on an EBS, so it is mutable, rather than in the immutable directory of the default Postgres image. This is mainly determined by the Kubernetes security context `readOnlyRootFilesystem`.